Automattic/jetpack
# Pocket Casts: Add Pocket Casts block for internal dev purposes.

#### Changes proposed in this Pull Request:
This PR adds a Pocket Casts block variation for **internal dev purposes**.

This allows the Pocket Casts team to test this block against their oEmbed endpoint which is still under development.

#### Jetpack product discussion
pdDOJh-bf-p2 & pdeCcb-dg-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

1. Apply patch D77986-code.
2. Add `add_filter( 'jetpack_block_editor_pocket_casts_feature', '__return_true' );` to your config file. (See D77986-code for more details).
3. Open up block editor.
4. Add "Pocket Casts" block to your editor.
5. Pasting a pocket cast link will not work at the moment as the oEmbed service is still under development.
 
![image](https://user-images.githubusercontent.com/1287077/161501996-1ed8e694-3a12-43c1-9b12-d28c2518f3dd.png)

![image](https://user-images.githubusercontent.com/1287077/161502003-7a1ef463-6add-4e91-a7bf-20ed0805bb35.png)
